website/content/docs/job-specification/affinity.mdx

```diff
@@ -89,10 +89,11 @@ allocations.
   For a detailed explanation of these values and their behavior, please see
   the [operator values section](#operator-values).
 
-- `value` `(string: "")` - Specifies the value to compare the attribute against
-  using the specified operation. This can be a literal value, another attribute,
-  or any [Nomad interpolated
+- `value` `(string: <required>)` - Specifies the value to compare the attribute
+  against using the specified operation. This can be a literal value, another
+  attribute, or any [Nomad interpolated
   values](/nomad/docs/reference/runtime-variable-interpolation#interpreted_node_vars).
+  The `value` field is required.
```

Comment:
I thought this conflicted with the other doc, but I see from the `operator` section above what you meant by "affinities don't support all the same operators" -- `affinity` doesn't have `is_set` or any others that allow for empty `value`s